dir.props

```diff
@@ -128,9 +128,11 @@
     <TargetsWindows Condition="'$(BuildOS)' == 'Windows_NT'">true</TargetsWindows>
 
     <TargetsUnix Condition="'$(TargetsFreeBSD)' == 'true' or '$(TargetsLinux)' == 'true' or '$(TargetsNetBSD)' == 'true' or '$(TargetsOSX)' == 'true'">true</TargetsUnix>
+    <TargetsLinuxCross Condition="'$(TargetsLinux)' == 'true' and '$(CROSSCOMPILE)' != ''">true</TargetsLinuxCross>
```

Comment:
If the environment variable '$__DistroRid' is not used in other place, we can overwrite the value of `$__DistroRidCross` to `$__DistroRid` in build.sh. Then we don't need `TargetLinuxCross`.